% Generated by roxygen2 (4.1.1): do not edit by hand
% Please edit documentation in R/functools.R
\name{Repeat}
\alias{Repeat}
\title{Repeat}
\usage{
Repeat(f, x)
}
\arguments{
\item{f}{a predicate function.}
}
\value{
the negation of that function.
}
\description{
\code{Repeat()} turns a function that throws an error into a function that returns a default value when there’s an error.
The essence of failwith() is simple; it’s just a wrapper around try(), the function that captures errors and allows execution to continue.
}
\examples{
Some examples:
}

